tests/integration/test_rest_consumer.py

```diff
@@ -87,7 +88,7 @@ async def test_subscription(rest_async_client, admin_client, producer, trail):
     topic_name = new_topic(admin_client)
     instance_id = await new_consumer(rest_async_client, group_name, fmt="binary", trail=trail)
     sub_path = f"/consumers/{group_name}/instances/{instance_id}/subscription{trail}"
-    consume_path = f"/consumers/{group_name}/instances/{instance_id}/records{trail}?timeout=1000"
```

Comment:
interesting, why do we want to increase the default timeout? Have you noticed that the performance are degratating or there is another rationale behind this?

Reply:
Because of the topic-pattern subscription later in this test we need to give the consumer a bit more time to discover everything and actually get assigned the topics.